duecredit/collector.py

```diff
@@ -161,8 +161,12 @@ def __contains__(self, entry):
             return entry.path.startswith(self.path + '.')
 
     @property
+    def entry_key(self):
+        return self.entry.get_key()
```

Comment:
Do we need it?
Why not to have key as a property in entry and then use `entry.key`